docs/week-3.md

```diff
@@ -0,0 +1,568 @@
+properly. Do not worry because Material UI is here to save the day!
+
+### What is Material UI?
+
+Material UI is a popular framework in React. It is a library filled with pre-styled components that are super
+helpful. There are many different components that basically replace the need for any pure html components in
+your project. Each of the pre-made components will save you a ton of time on styling and formatting.
+
+### How can I use it?
+
+All you have to do is import any component you want from the list found 
+[here](https://material-ui.com/components/box/). For example let us replace the html `<button>` for John's 
+likes with a Material UI Button component.
+
+```
+import React, { useState } from 'react';
+import LuckyNum from './LuckyNum';
+import { Button } from '@material-ui/core'
+
+const Profile = () => (
+
+    // initialize the state to 0 because we start with 0 likes
+    const [profileLikes, setProfileLikes] = useState(0);
+
+    // handles incrementing the likes of John's profile
+    function handleLikes {
+        setProfileLikes(profileLikes + 1);
+    }    
+
+    <div>
+        <img src="profileImage.jpg"/>
+        <h1>Hi, my name is John! My favorite color is blue.</h1>
+        <LuckyNum />
+        <Button onClick={()=>handleLikes}>Likes: {profileLikes}</Button>
+    </div>
+);
+
+export default Profile;
+```
+
+### Styling Material UI Components
+If you wanted to style the components in Material UI a little more to tailor your specific needs you can do that!
+For example let us make the newly imported Button component on John's profile to be black.
+
+We first have to import `makeStyles` from `@material-ui/core/styles`. We then make an object for each class we 
+want to apply to our components. We make a lambda function called `useStyles` that is equal to `makeStyles`
+and pass in our objects to the `makeStyles` function as one big object. We then call `useStyles` in our 
+component to apply in `className`.
+
+```
+import React, { useState } from 'react';
+import LuckyNum from './LuckyNum';
+import { Button } from '@material-ui/core'
+
+const useStyles = makeStyles({
+    
+    makeButtonBlack: {
+        color: 'black',
+        font: '30px'
+    }
+})
+
+const Profile = () => (
+
+    // allows us to use our custom styles
+    const classes = useStyles();
+
+    // initialize the state to 0 because we start with 0 likes
+    const [profileLikes, setProfileLikes] = useState(0);
+
+    // handles incrementing the likes of John's profile
+    function handleLikes {
+        setProfileLikes(profileLikes + 1);
+    }    
+
+    <div>
+        <img src="profileImage.jpg"/>
+        <h1>Hi, my name is John! My favorite color is blue.</h1>
+        <LuckyNum />
+        <Button 
+                className={classes.makeButtonBlack} 
+                onClick={()=>handleLikes}>Likes: {profileLikes}
+        </Button>
+    </div>
+);
+
+export default Profile;
+```
+
```

Comment:
Add a conclusion to your story! What happened with John and Doug?? Did they complete their website???????